SEED-platform/seed
# Lint fixes

#### What's this PR do?
- Adds html/js/scss lint checking to the CI
- Removes the _legacy_ inventory list page
- Fixes prettier formatting for html files
- Fixes eslint errors in js files
- Fixes stylelint errors in scss files
- Fixes typos

#### How should this be manually tested?
Monkey test, check that everything works as expected

To review (and automatically fix) lint errors you can use the following commands.
##### From Python
`tox -e lint` and `tox -e lint-fix`

##### From Node
`npm run lint` and `npm run lint:fix`

or, you can run the individual lint targets:

- `npm run eslint` and `npm run eslint:fix`
- `npm run prettier` and `npm run prettier:fix`
- `npm run stylelint` and `npm run stylelint:fix`
